### PR TITLE
Refine userguide

### LINKED ISSUE #102: Reformat UG

### PR DESCRIPTION
**Pull request by V4Vern**
fixes #102

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -501,4 +544,62 @@ Example of usage:
...
+| Adding an exercise log    			| program /log <EXERCISE_NAME [string]> /weight <WEIGHT [integer]> /sets <NUMBER_OF_SETS [integer]> /reps <NUMBER_OF_REPS [integer]>   | program /log benchpress /weight 125 /sets 3 /reps 5    |
+| Adding an exercise log for a separate date    			| program /log <EXERCISE_NAME [string]> /weight <WEIGHT [integer]> /sets <NUMBER_OF_SETS [integer]> /reps <NUMBER_OF_REPS [integer]> /date <DATE [yyyy-mm-dd]>   | program /log benchpress /weight 125 /sets 3 /reps 5 /date 2024-03-25    |
+| Viewing historic logs     	 		| program /history      							     | program /history      			       |
+| Viewing historic logs     	 		| program /history <DATE [yyyy-mm-dd]>      					     | program /history <DATE [yyyy-mm-dd]>   	       |
+| Displaying Help Menu Category: Exercise    	| help /exercise <INDEX [integer]>       					     | help /exercise 3     			       |
+| Displaying Help Menu Category: Workout    	| help /workout  <INDEX [integer]>     						     | help /workout 4     			       |
+| Displaying Help Menu Category: Program    	| help /program  <INDEX [integer]>        					     | help /program 7      			       |
+
+
```
Comment (LWachtel1) [inline docs/UserGuide.md:605]: Great additions to UG!

**Review by LWachtel1 — COMMENTED** [review]
Overall, LGTM. Should Help Menu have a link at the start?

File: docs/UserGuide.md (hunk 1 above)
Comment (V4Vern) [reply docs/UserGuide.md:605]: oops I forgot about the help menu link! Thanks for the reminder

File: docs/UserGuide.md [hunk 2]
```diff
@@ -30,7 +32,14 @@ Welcome to BYTE-CEPS, your CLI-based all-in-one tool for setting and tracking fi
 		- [Adding an exercise log for a separate date](#adding-an-exercise-log-for-a-separate-date)
 		- [Viewing logs](#viewing-logs)
 		- [Viewing historic logs](#viewing-historic-logs)
-
+  	- [Help Menu](#help-menu)
+  		- [Displaying Help Menu Category: Exercise](#displaying-help-menu-category-exercise)
+  	 	- [Displaying Help Menu Category: Workout](#displaying-help-menu-category-workout)
+  	  	- [Displaying Help Menu Category: Program](#displaying-help-menu-category-program)
```
Comment (LWachtel1) [inline docs/UserGuide.md:38]: LGTM

**Review by LWachtel1 — APPROVED** [review]
LGTM

**Pull request merged**